onnx2kerastl/operation_layers.py

```diff
@@ -223,9 +220,8 @@ def convert_sqrt(node, params, layers, lambda_func, node_name, keras_name):
         assert AttributeError('More than 1 input for sqrt layer.')
 
     input_0 = ensure_tf_type(layers[node.input[0]], name="%s_const" % keras_name)
+    layers[node_name] = K.sqrt(input_0)
 
-    sqrt_layer = OnnxSqrt(name=keras_name)
-    layers[node_name] = sqrt_layer(input_0)
```

Comment:
I would keep `OnnxSqrt` just because it looks better than another TFOPLambda and we already made the effort to create it

Reply:
OK